spec/features/placeholder_users/create_spec.rb

```diff
@@ -29,34 +29,60 @@
 require 'spec_helper'
 
 describe 'create placeholder users', type: :feature, selenium: true do
-  let(:current_user) { FactoryBot.create :admin }
   let(:new_placeholder_user_page) { Pages::NewPlaceholderUser.new }
 
-  before do
-    allow(User).to receive(:current).and_return current_user
-  end
+  shared_examples_for 'placeholders creation flow' do
+    context 'with enterprise', with_ee: %i[placeholder_users] do
+      it 'creates the placeholder user' do
+        visit new_placeholder_user_path
+
+        new_placeholder_user_page.fill_in! name: 'UX Designer'
+
+        perform_enqueued_jobs do
```

Comment:
I don't understand why any jobs would get triggered when creating a placeholder user. Is that old copy&pasted code that I added there?

Reply:
Yes I just moved that code, not sure why this was added